tests/test_screendoor.py

```diff
@@ -1,19 +1,115 @@
 """Tests for screendoorapi package"""
+import json
+from unittest.mock import patch
 from screendoor_sdk.screendoor import Screendoor
 
-def test_screendoor_url():
-    """Test screen api url"""
-    sd_key = 'SCREENDOOR_API_KEY'
-    sd_project_id = '1234'
+SD_KEY = 'SCREENDOOR_API_KEY'
+SD_PROJECT_ID = '1234'
+
+MOCK_RESPONSES = """[
+    {
+        "id": 11111,
+        "sequential_id": 1,
+        "project_id": 1,
+        "form_id": 1,
+        "initial_response_id": null,
+        "pretty_id": "ABCDEFG",
+        "submitted_at": "2019-01-02T00:00:00.000Z",
+        "responses": {
+            "hkf58ruh": {
+                "checked": [
+                    "Chocolate Chip"
+                ]
+            },
+            "e88dqmqb": null
+        },
+        "rating_aggregates": {},
+        "average_rating": null,
+        "num_ratings": 0,
+        "created_at": "2019-01-01T00:00:00.000Z",
+        "updated_at": "2019-01-02T00:00:00.000Z",
+        "status": "Triage",
+        "labels": [],
+        "responder_language": "en",
+        "responder": {
+            "name": "Test One",
+            "email": "test.one@test.com"
+        },
+        "deleted_at": null,
+        "submission_source": {
+            "type": "frontend",
+            "hostname": "forms.fm"
+        }
+    }
+]"""
+
+
+
+def test_get_url():
+    """ Test get api url """
     params = {'per_page': 50, 'page' : 10}
 
-    sd_api = Screendoor(sd_key, '0', 'https://screendoor.dobt.co/api')
-    sd_url = sd_api.get_url({
-        'path' : '/projects/' + sd_project_id + '/responses',
+    scrndr = Screendoor(SD_KEY, '0', 'https://screendoor.dobt.co/api')
+    sd_url = scrndr.get_url({
+        'path' : '/projects/' + SD_PROJECT_ID + '/responses',
         'params' : params
     })
 
     expected_sd_url = 'https://screendoor.dobt.co/api/projects/'
-    expected_sd_url += sd_project_id+'/responses?v=0&api_key='+sd_key+'&per_page=50&page=10'
+    expected_sd_url += SD_PROJECT_ID+'/responses?v=0&api_key='+SD_KEY+'&per_page=50&page=10'
 
     assert expected_sd_url == sd_url
+
+def test_project_labels():
+    """ Test project_labels """
+    labels = [
+        {"name": "Missing information", "color": "3600ff", "responses_count": 1},
+        {"name": "Questionable", "color": "336699", "responses_count": 0}]
+
+    scrndr = Screendoor(SD_KEY)
+    with patch('screendoor_sdk.screendoor.requests.get') as mock_get:
```

Comment:
these mock_gets don't actually make a call to screen door?
so if screendoor makes a change to their api, these test will not catch it.

Reply:
Yeah, that's exactly the goal of using mocks in tests. The goal of the tests is to test the code within the SDK itself, not testing the actual screendoor API.   Another example would, what if the Screendoor API went down for some reason, if we are testing the actual live API, these tests would fail also, so it fails CI, does it mean our code is bad? 

I posted this on slack before: https://circleci.com/blog/testing-3rd-party-apis-and-microservices/ 

I do agree we should have something to test 3rd party APIs, for areas such as consistency, availability, etc. Perhaps we can discuss in our eng meeting? @sashax